docker/scripts/init-shard.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash 
 
+readarray -d . -t verarr <<< "${VERSION}"
+readarray -d : -t version <<< "${verarr[0]}"
+echo "Mongo version: ${version[1]}"
+
+if ((${version[1]} <= 4))
+then
+  echo -e 'mongo' > /usr/bin/mongosh && chmod +x /usr/bin/mongosh
```

Comment:
Maybe it's better to use this approach https://github.com/supercharge/mongodb-github-action/blob/main/start-mongodb.sh#L11-L30

Reply:
Yes this looks better, ok I will change it.